docs/4.1/components/pagination.md

```diff
@@ -15,36 +15,34 @@ In addition, as pages likely have more than one such navigation section, it's ad
 {% capture example %}
 <nav aria-label="Page navigation example">
   <ul class="pagination">
-    <li class="page-item"><a class="page-link" href="#">Previous</a></li>
+    <li class="page-item"><a class="page-link" href="#" rel="prev">Previous</a></li>
```

Comment:
only information on these `rel="..."` attributes being of some use i came across in the past related to them being used on `<link rel="..." ...>` elements in the `<head>`. e.g. https://support.google.com/webmasters/answer/1663744?hl=en

if authors want to add this to their code, cool (assuming the attributes on actual links, rather than `<link...>` elements, does anything). but i don't think it's necessary for our example code